src/types/zetkin.ts

```diff
@@ -161,6 +161,11 @@ interface ZetkinQuestion {
     response_type: RESPONSE_TYPE;
 }
 
+export interface ZetkinOption {
```

Comment:
I would rename this to something like `ZetkinSurveyOption` which is less ambiguous.